docs/root/configuration/http_conn_man/stats.rst

```diff
@@ -55,6 +55,7 @@ statistics:
    downstream_rq_time, Histogram, Request time milliseconds
    downstream_rq_idle_timeout, Counter, Total requests closed due to idle timeout
    downstream_rq_overload_close, Counter, Total requests closed due to envoy overload
+   downstream_rq_path_timeout, Counter, Total requests closed due to request path timeout
```

Comment:
It's a good idea to be careful about consistent nomenclature.  It looks as if this (the request path timeout) is the same as what's configured above as stream_request_timeout.  Could you either call them the same thing, or include a comment as to their relationship (if they have one)?